src/status_im/ui/screens/accounts/events.cljs

```diff
@@ -150,7 +150,7 @@
 (handlers/register-handler-fx
   :account-finalized
   (fn [{db :db} _]
-    {:db db
+    {:db (dissoc db :accounts/create)
```

Comment:
we can't dissoc `:accounts/create` because we use `:accounts/create :show-welcome?`